README.rst

```diff
@@ -55,3 +55,25 @@ functions.
 Development
 ***************
 We follow this `git branching model <http://nvie.com/posts/a-successful-git-branching-model/>`_, please have a look at it.
+
+
+*******
+Install Instructions (from pip's database)
+*******
+pip install pybloom-live
```

Comment:
Rather than using plain text, it would be nicer to use markdown syntax, so it will look like `pip install pybloom-live`.